QTMSettings.cs

```diff
@@ -1185,7 +1058,7 @@ internal static Settings6DOF ConvertToSettings6DOF(Settings6DOF_V2 settings6DOF)
         public bool Enabled;
```

Comment:
We don't need an **XmlElement("Enabled")** here as well? Just double checking since it seems that a lot of variables have a corresponding XmlElement.

Reply:
If you expand the collapsed code you will see that there is an `XmlElement("Enabled")` there :)